### PR TITLE
Regular updates (May 11)

### LINKED ISSUE #1301: cmd ref: document new subcommand `remote rename`
Related to issue [iterative/dvc#3599](https://github.com/iterative/dvc/issues/3599).

### LINKED ISSUE #1314: coding style guidelines missing from Docs and Website Page
The link in the text in `coding style guidelines` on the page [Docs and Website page](https://dvc.org/doc/user-guide/contributing/docs) which ideally should list principles and guidelines seems to be a dead in-page link. 

The referred link is https://dvc.org/doc/user-guide/contributing/docs#code-style-guidelines on clicking on the text, but it seems that there is no such section marked on that markdown file.

There is a mention of Prettier's usage for code formatting, but that has been put under [Development Environment](https://dvc.org/doc/user-guide/contributing/docs#development-environment).
Maybe we can add another section named as `Code Style Guidelines`, just like [Doc Style Guidelines](https://dvc.org/doc/user-guide/contributing/docs#doc-style-guidelines-javascript-and-markdown), moving the info and steps related to Prettier into it, and refer it through `coding style guidelines`

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- A few more changes to plot cmd refs (matched to core repo) — see iterative/dvc/pull/3759
- GSoD blog post update
- Misc. stuff e.g. https://github.com/iterative/dvc.org/pull/1268#pullrequestreview-417252532
- Closes #1314
- Finish #1318 to close #1301

### CODE REVIEW

File: content/docs/command-reference/metrics/diff.md [hunk 1]
```diff
@@ -61,6 +60,8 @@ They're calculated between two commits (hash, branch, tag, or any
   path is allowed. It will override `xpath` defined in the corresponding
   DVC-file. See `dvc metrics show` for more details.
 
+- `--all` - list all metrics, even those without differences to compare.
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/diff.md:63]: those that haven't changed? (hard to understand what does the option mean)

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/diff.md:63]: Updated

File: content/blog/2020-04-30-gsod-ideas-2020.md [hunk 2]
```diff
@@ -154,6 +154,21 @@ technical writer, [Jorge](https://github.com/jorgeorpinel).
...
+   with proven experience in SEO or SRE!
+
+   What tools should we employ? (e.g. Google Analytics, etc.) What trends and
+   reports do we need to focus on? What kinds of users do we have and what
+   interaction flows do they each follow? Can we semi-identify these users
+   and/or cross-examine their data with DVC
+   [usage analytics](https://dvc.org/doc/user-guide/analytics)? Let's come up
+   with a plan to answer these and other related questions!
+
+   _Difficulty rating:_ Beginner-Medium<br/><br/>
```
Comment (jorgeorpinel) [inline content/blog/2020-04-30-gsod-ideas-2020.md:170]: Per
> I thought maybe another good project idea is to improve on our SEO/analytics? Some of the candidates may have worked/had internships on that topic.

File: content/blog/2020-04-30-gsod-ideas-2020.md [hunk 3]
```diff
@@ -154,6 +154,21 @@ technical writer, [Jorge](https://github.com/jorgeorpinel).
 
    _Difficulty rating:_ Beginner-Medium<br/><br/>
 
+1. **SEO/ Site Analytics.** Our current website analytics are somewhat basic. We
+   will need to have a clear strategy to follow and improve our Search Engine
+   results, as well as to understand the behavior of our users, and this is a
+   priority for 2020. This should be a relatively simple project for someone
+   with proven experience in SEO or SRE!
```
Comment (shcheklein) [inline content/blog/2020-04-30-gsod-ideas-2020.md:161]: SRE stands for Site Reliability Engineer these days :)

Comment (shcheklein) [reply content/blog/2020-04-30-gsod-ideas-2020.md:161]: we can link a few tickets we have - review metas, improve metas, SEO ?

Comment (shcheklein) [reply content/blog/2020-04-30-gsod-ideas-2020.md:161]: Le's move it a bit down? it's a bit specific ...

File: content/docs/command-reference/metrics/diff.md [hunk 4]
```diff
@@ -61,6 +60,8 @@ They're calculated between two commits (hash, branch, tag, or any
   path is allowed. It will override `xpath` defined in the corresponding
   DVC-file. See `dvc metrics show` for more details.
 
+- `--all` - list all metrics in the comparison, even those without changes.
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/diff.md:63]: `in the comparison` feels like we can omit this?

File: content/docs/command-reference/remote/modify.md [hunk 5]
```diff
@@ -138,13 +138,13 @@ these settings, you could use the following options:
   $ dvc remote modify myremote sse AES256
   ```
 
-- `sse_kms_key_id` - SSE-KMS key to use to encrypt data uploaded to S3, normally
-  specified when the `sse` parameter is set to `aws:kms`. This parameter will be
-  passed directly to AWS's S3 functions, so DVC supports any value that AWS
-  supports, including both KMS key ids and aliases.
+- `sse_kms_key_id` - identifier of the key to encrypt data uploaded when using
+  SSE-KMS.. Required when the `sse` parameter (above) is set to `aws:kms`. This
```
Comment (shcheklein) [inline content/docs/command-reference/remote/modify.md:142]: .. -> .?

File: content/docs/command-reference/run.md [hunk 6]
```diff
@@ -256,14 +256,21 @@ processing:
...
           -p seed,train.lr,train.epochs
           python train_model.py matrix-train.p model.p
 ```
 
+Note that DVC doesn't pass the parameter values to the command being run. The
+program (`train_model.py` in this case) will have to open and parse
+`params.yaml` by itself and use the params specified above
+(`seed,train.lr,train.epochs`). DVC will keep an eye on these param values (as
+well as on the regular dependency files) and know that the stage should be
+reproduced if/when they change.ee `dvc params` for more details.
```
Comment (shcheklein) [inline content/docs/command-reference/run.md:272]: `..ee` - typo?

File: content/blog/2020-04-30-gsod-ideas-2020.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/blog/2020-04-30-gsod-ideas-2020.md:161]: > SRE stands for Site Reliability Engineer these days :)

Yes, that's what I meant. Is it inaccurate to mention that role?

Comment (shcheklein) [reply content/blog/2020-04-30-gsod-ideas-2020.md:161]: yes, I think it's not very relevant here (SRE is close to DevOps, not sure why would it be relevant with SEO and meta tags)

Comment (jorgeorpinel) [reply content/blog/2020-04-30-gsod-ideas-2020.md]: True. I was confused... I think I meant QA. OK, updated in 592bc548!

**Comment by jorgeorpinel** [comment]
- [ ] I'll also merge #1318 right after this one.

**Comment by jorgeorpinel** [comment]
Please ignore failed test, it's just the link to the new cmd ref (`dvc remote rename`).

**Comment by jorgeorpinel** [comment]
Thanks for approval, I just updated the indentation in some code examples in fb498c2, merging now 🙂

**Pull request merged**